apps/core/scenes/passengerForm/BaggageBundles/index.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+
+import * as React from 'react';
+import { Card } from '@kiwicom/universal-components';
+import { HeaderWithIcon } from '@kiwicom/margarita-components';
+
+import Header from './Header';
+import BaggageBundle from './BaggageBundle';
+import { bags } from './BagTypes';
+
+const data = [
+  {
+    type: 'cabin',
```

Comment:
What is this name and what is the difference between type here and in `bundles`? I would keep naming in the same way if it describes the same thing. What do you think?

Reply:
I am not sure about this one, we have `cabin` and `checked` bundle groups in design. So idk how it can renamed